lib/external_data_sets_two_electric_boogaloo.rb

```diff
@@ -0,0 +1,49 @@
+class FacilityObjects
+  def initialize
+    @facilities_list = []
+  end
+  
+  def create_facilities(api_data)
+    api_data.each do |facility_data|
+      fac_data = dmv_state_check(facility_data)
+      facility = Facility.new(fac_data)
+      @facilities_list << facility
+    end
+    @facilities_list
+  end
```

Comment:
Similar to my above comment.. you can probably do away with the `@facilities_list` in your initialize method, and instead use a `map in your `create_facilities` method.